prog.py

```diff
@@ -41,15 +46,18 @@ def write_to_file(toAdd, file_name):
 def main():
     buffer, parsed = read_from_file('messages.txt')
     print(buffer)
-    for date, text in parsed:
-        print(date.strftime("%d.%m.%Y %H:%M:%S"), ": ", text)
-
-    while True:
-        # ask for new input
-        toAdd = user_input()
-        # string to write
-        if toAdd:
-            write_to_file(toAdd, 'messages.txt')
+    for phone_number, message in parsed:
+        print(phone_number, ": ", message)
+
+    # while True:
+    #     # ask for new input
+    try: user_input()
```

Comment:
The expected program flow must be like:
* (1) user enters a phone number
* if it is correct, then user proceeds to entering a message
* otherwise, go to step 1

Currently, it would 
* ask for a phone number,
* if it's valid, ask for a message; if not, go to next step
* ask for a phone number again
* if it's not valid - break the whole program by raising ValueError,
* otherwise, ask for a message (possibly - once more, if there was no error on step 1

The correct behavior is usually achieved by having a `while` loop which repeats the input until the provided value is valid.